include/swift/AST/ArchetypeBuilder.h

```diff
@@ -466,6 +468,18 @@ class ArchetypeBuilder::PotentialArchetype {
     EquivalenceClass.push_back(this);
   }
 
+  /// \brief Construct a new potential archetype for a type alias.
+  PotentialArchetype(PotentialArchetype *Parent, TypeAliasDecl *TypeAlias)
+  : ParentOrParam(Parent), NameOrAssociatedType(TypeAlias),
+  Representative(this), IsRecursive(false), Invalid(false),
+  SubstitutingConcreteType(false), RecursiveConcreteType(false),
+  RecursiveSuperclassType(false), Renamed(false)
```

Comment:
Style: please indent `:` and align the members (or just run clang-format on it).